app/views/users/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
For a simple form like this, we'd do something like:

```<!--Descriptive page name, messages and instructions-->
<h2 class="heading"><%= ts('Resend Confirmation Instructions') %></h2>
<!--/descriptions-->

<!--subnav-->
<!--/subnav-->

<!--main content-->
<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post, class: "simple post" }) do |f| %>
  <%= devise_error_messages! %>

  <fieldset>
    <p>
      <%= f.label :email %>
      <%= f.email_field :email, autofocus: true, value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %> 
      <%= f.submit ts('Resend Confirmation Instructions') %>
    </p>
  </fieldset>
<% end %>
<!--/content-->```

(I've left off `<%= render "devise/shared/links" %>` because there doesn't appear to be a partial by that name.)